docs/source/api/jobs_id.rst

```diff
@@ -18,6 +18,7 @@
 ***************
 ``jobs/{{ID}}``
 ***************
+.. deprecated:: ATCv4
 .. caution:: In the vast majority of cases, it is preferred to use the ``id`` query parameter of the :ref:`to-api-jobs` endpoint instead.
```

Comment:
You can collapse these, they're not really different admonitions. Use other thing because this is deprecated.

Reply:
done